tools/docker-httpbin-override.yml

```diff
@@ -0,0 +1,11 @@
+version: '2'
```

Comment:
I think we lost a commit somewhere - if I recall correctly we removed this

Reply:
fixed https://github.com/ansible/awx/pull/4124/commits/34fa5266905ce15e7e9d9bcb44cf03a4ff10c049